### PR TITLE
doc: update 'asa.rst' for 2.6 release

### PR DESCRIPTION
**Pull request by yonghuah**
Update security vulnerability fix in 2.6 release.

Signed-off-by: Yonghua Huang <yonghua.huang@intel.com>

### CODE REVIEW

**Comment by yonghuah** [comment]
@NanlinXie  pls review, thanks.

**Review by NanlinXie — APPROVED** [review]

File: doc/asa.rst [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+Addressed in ACRN v2.6
+************************
+
+We recommend that all developers upgrade to this v2.6 release (or later), which
+addresses the following security issues that were discovered in previous releases:
+
+-----
+
+-  Memory leakage vulnerability in ``devicemodel/hw/pci/xhci.c``
+    De-initializing of emulated USB devices result in memory leakage issue
```
Comment (fitchbe) [inline doc/asa.rst:15]: "result" --> "results"
"in memory leakage issue" --> "in a memory leakage issue" _or_ "in a memory leak"

**Review by fitchbe — APPROVED** [review]

File: doc/asa.rst (hunk 1 above)
Comment (dbkinder) [reply doc/asa.rst]: I fixed this and pushed the change back to yonghuah's branch

**Review by dbkinder — APPROVED** [review]
I addressed @fitchbe comments

**Pull request merged**